exts/rag_bge_small_en_v15/src/lib.rs

```diff
@@ -166,7 +166,8 @@ pub extern "C-unwind" fn background_main(arg: pg_sys::Datum) {
             Server::builder()
                 .add_service(EmbeddingGeneratorServer::new(embedder))
                 .serve_with_incoming_shutdown(uds_stream, async {
-                    while BackgroundWorker::wait_latch(None) {
+                    while BackgroundWorker::wait_latch(Some(Duration::from_secs(0))) {
```

Comment:
Can you please add a comment here about what Duration::from_secs(0) means i.e. "this makes the background worker to poll instead of blocking. Blocking is not an option because <...>"

Reply:
Same for the latter